packages/application-shell/src/components/application-shell/application-shell.tsx

```diff
@@ -404,10 +424,16 @@ export const RestrictedApplication = <
                               `}
                             >
                               <PortalsContainer
+                                // @ts-ignore
+                                ref={layoutRefs}
                                 offsetTop={DIMENSIONS.header}
-                                offsetLeft={DIMENSIONS.navMenu}
+                                offsetLeft={
+                                  projectKeyFromUrl ? DIMENSIONS.navMenu : '0px'
+                                }
```

Comment:
I would like to ask about this check for `projectKeyFromUrl`.
What use case is this?

Reply:
In the "account" app there is no left side menu. We can derive this check from the presence of the project key in the url, or not.